app/javascript/components/MetricsPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom'
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography'
+import Card from '@mui/material/Card'
+import { createTheme, ThemeProvider, styled } from '@mui/material/styles';
+import SurveyList from './SurveyList';
+import SurveyListActions from './SurveyListActions'
+
+
+const darkTheme = createTheme({ palette: { mode: 'dark' } });
```

Comment:
this stuff isnt needed and some of the imports are not needed, we should remove those